docs/resources/mws_log_delivery.md

```diff
@@ -56,18 +56,21 @@ resource "aws_s3_bucket_policy" "logdelivery" {
 }
 
 resource "databricks_mws_credentials" "log_writer" {
+    provider         = databricks.mws
```

Comment:
please revert this file, as it has no provider definition with this alias. also it might be a separate module, so it's a completely valid doc.

```
provider "databricks" {
  alias = "mws"
}
```

Reply:
Done.